README.md

```diff
@@ -37,10 +37,10 @@ All the Valkey commands are in the Commands folder of the Valkey target. These a
 
 ### Pipelining commands
 
-In some cases it is desirable to send multiple commands at one time, without waiting for the response after each command. This is called pipelining. You can do this using the function `pipeline(_:)`. This takes a parameter pack of commands and returns a parameter pack with the responses once all the commands have executed.
+In some cases it is desirable to send multiple commands at one time, without waiting for the response after each command. This is called pipelining. You can do this using the function `execute(_:)`. This takes a parameter pack of commands and returns a parameter pack with the responses once all the commands have executed.
```

Comment:
we should call out that execute supports more than one command and automatically pipelines in those cases.

Reply:
I thought that was what the text said, anyway I edited it slightly to make it more obvious